Lib/test/test_importlib/test_windows.py

```diff
@@ -112,7 +112,7 @@ def test_module_not_found(self):
 class WindowsExtensionSuffixTests:
     def test_tagged_suffix(self):
         suffixes = self.machinery.EXTENSION_SUFFIXES
-        abi_flags = "t" if sysconfig.get_config_var("Py_GIL_DISABLED") else ""
+        abi_flags = "t" if support.Py_GIL_DISABLED else ""
```

Comment:
@vstinner  - looks like Windows tests are failing (possibly due to missing import for support):

https://github.com/python/cpython/actions/runs/7090430380/job/19297369118?pr=112709

Reply:
Ah right,  I saw it before. Thanks for the reminder. It should now be fixed.